src/main/resources/assets/computercraft/lua/rom/help/speakers

```diff
@@ -2,7 +2,7 @@ The Speaker is a peripheral device available for CraftOS. Type "help peripheral"
 
 Methods exposed by the Speaker:
 playSound( sResourceName, nVolume, nPitch )
-playNote( sInstrumentName, nVolume, nTicks )
+playNote( sInstrumentName, nVolume, nPitch )
```

Comment:
As I recall, internally it uses the term pitch for the function in the code thus what lead to that change. According to #247 it goes by right clicks, so I assume this is correct and the function in the code just needs it's variable name changed to prevent confusion.